asdf_astropy/converters/unit/quantity.py

```diff
@@ -27,4 +27,4 @@ def from_yaml_tree(self, node, tag, ctx):
             # and documentation.
             value = value._make_array()
 
-        return Quantity(value, unit=node["unit"])
+        return Quantity(value, unit=node["unit"], copy=False)
```

Comment:
Do we also need `subok`? Maybe @mhvk can confirm.

```suggestion
        return Quantity(value, unit=node["unit"], copy=False, subok=True)
```

Reply:
Correct, this will only be handling the two types listed here: https://github.com/WilliamJamieson/asdf-astropy/blob/3888a86746953afee37b226c73ff96f22e7e2dea/asdf_astropy/converters/unit/quantity.py#L9-L12

Reply:
Soon we'll need to serialize `coordinates.SpectralQuantity` but I guess all we need to do then is add it to the above list.